build.gradle

```diff
@@ -0,0 +1,21 @@
+apply plugin: 'java'
+apply plugin: 'war'
+
+repositories {
+    jcenter()
+    mavenCentral()
+    mavenLocal()
+}
+
+dependencies {
+    compile fileTree(dir: 'lib', include: '*.jar')
+    compile 'javax.servlet:javax.servlet-api:3.0.1'
+    compile 'postgresql:postgresql:9.1-901.jdbc4'
+
+    testCompile 'simple-jndi:simple-jndi:0.11.4.1'
+    testCompile 'junit:junit:4.12'
+}
+
+sourceSets.main.java.srcDirs = ["src"]
+// the tests fail because they have environemnt specific parameters
+// sourceSets.test.java.srcDirs = ["test"]
```

Comment:
I am currently looking at the Gradle documentation so that having a Gradle build file having the same functionalities as my ANT scripts (or using directly them). Your Gradle build is only valid for the TAP Library ; I would like to extend it to take into account the ADQL and UWS libraries as well.

About the failing tests, I have the explanation: in my ANT script, I do not run all JUnit tests (I skip those requiring environment specific parameters), whereas Gradle is executing all of them. However, I am also currently working to have an embedded testing database so that all JUnit tests can pass on any environment.

In brief, I will probably fetch your `build.gradle` in this Pull Request, and later I will extend it with the functionalities I mentioned above.

Thank you for this suggestion of using Gradle :-)